### DISCUSSION TITLE
Thumbnails and Previews

### DISCUSSION THREAD

**Discussion by jmagly** [teia-community/teia-ui#27] [Ideas]
Would really love to see a thumbnail generator and the ability to show a preview when using links like this on social platforms. This is something myself and others have come to use object for linking often because it does have a thumbnail cache. 

![Screenshot 2022-03-19 181405](https://user-images.githubusercontent.com/1159087/159140278-6d9d50a9-a2d2-4e6f-89a6-bdf285926b96.png)

**Comment by melMass** [comment]
Definitely I know that @kylegrover looked into it!
@xat was proposing nextjs as it's possibly doable with it. Not sure about the scalability.
